src/services/Admin/index.js

```diff
@@ -45,177 +47,56 @@ const blacklistFragment = `fragment FortisDashboardView on BlacklistCollection {
                         }
                       }`;
 
-export const SERVICES = {
-    getDashboardSiteDefinition(translationLanguage, callback){
-        const query = ` ${AdminFragments.siteSettingsFragment}
-                        ${AdminQueries.getPipelineDenfintion}`;
-
-        const variables = { translationLanguage };
-        const host = process.env.REACT_APP_SERVICE_HOST
-        const POST = {
-            url: `${host}/api/settings`,
-            method: "POST",
-            json: true,
-            withCredentials: false,
-            body: { query, variables }
-        };
-
-        request(POST, callback);
-    },
-
-    fetchStreams(callback) {
-        const query = `
-        query Streams {
-          streams {
-            streams {
-              streamId
-              pipelineKey
-              pipelineLabel
-              pipelineIcon
-              streamFactory
-              params {
-                key
-                value
-              }
-              enabled
-            }
-          }
-        }`;
-  
-        const host = process.env.REACT_APP_SERVICE_HOST;
-        const POST = {
-          url: `${host}/api/settings`,
-          method: 'POST',
-          json: true,
-          withCredentials: false,
-          body: { query }
-        };
-  
-        request(POST, callback);
-      },
-
-      saveStreams(streams, callback) {
-      const query = `
-      mutation ModifyStreams($input: StreamListInput!) {
-        modifyStreams(input: $input) {
-          streams {
-            streamId
-            pipelineKey
-            pipelineLabel
-            pipelineIcon
-            streamFactory
-            params {
-              key
-              value
-            }
-            enabled
-          }
-        }
-      }
-      `;
+function fetchGqlData(endpoint, gqlQueryBody, callback) {
```

Comment:
This is pretty similar to [src/services/Dashboard/index.js:fetchGqlData](https://github.com/CatalystCode/project-fortis-interfaces/blob/5a22c53ad2baa561d134496f2185b2e0d5ff6c1f/src/services/Dashboard/index.js#L7-L20). Might be worth sharing that method.

Reply:
Yes definitely. Will do this in my next commit.